bench/irmin-pack/trace_definitions.ml

```diff
@@ -317,7 +317,7 @@ module Stat_trace = struct
 
     type config = {
       inode_config : int * int * int;
-      store_type : [ `Pack | `Pack_layered | `Pack_mem ];
+      store_type : [ `Pack | `Pack_mem ];
```

Comment:
This diff should be reverted in order to keep preserve the file format

Reply:
Done, thanks.